netbout-web/src/main/java/com/netbout/dynamo/DyAttachments.java

```diff
@@ -46,10 +46,11 @@
 
 /**
  * Dynamo attachments.
- *
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 2.0
+ * @todo #1090:30min No public static literals are allowed according to our
+ *  methodology. This class should be refactored to not contain any of them.
```

Comment:
@CarlosGines I think we should also rephrase this a little, saying "public static literals are not recommended" and I would also recommend adding the link to http://www.yegor256.com/2015/07/06/public-static-literals.html. Otherwise this might be very confusing :)